extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -27,6 +27,7 @@ import {
   DEFAULT_SAFEFRAME_VERSION,
   assignAdUrlToError,
 } from '../../amp-a4a/0.1/amp-a4a';
+import {is3pThrottled} from '../../amp-ad/0.1/concurrent-load';
```

Comment:
Need exception in dep-check-config.js

Reply:
...actually, it looks like this dependency was specifically blacklisted, as opposed to just not whitelisted. What's up with that?
```
  {
    filesMatching: 'extensions/**/*-ad-network-*.js',
    mustNotDependOn: [
      'extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js',
      'extensions/amp-ad/0.1/concurrent-load.js',
      'src/3p-frame.js',
      'src/iframe-helper.js',
    ],
  },
```

Reply:
Fixed thanks for running it down (removed concurrent-load from list... not sure why its there)